src/fides/cli/commands/view.py

```diff
@@ -27,12 +28,30 @@ def view_config(
     ctx: click.Context, section: str = "", exclude_unset: bool = False
 ) -> None:
     """
-    Prints the configuration values being used.
+    Prints the configuration values being used as a `toml` file.
     """
     config = ctx.obj["CONFIG"]
```

Comment:
Now that Adam has some work merged that allows you to override some of the config variables via the API, this wouldn't necessarily show the config value that's being used, as the API values override.

Reply:
Hmmmm, would a better solution here be to have a separate command that will hit the config API endpoint and show the configuration being used by the server? Sans sensitive values

It is quite possible and even expected the local configuration values for the CLI user will be different than the values that webserver is using

Reply:
For now I'm going to note this in the help string, but will hold off on adding that command until there is a concrete need for it

Reply:
I didn't realize the cli user config could be different than the CLI user so do what you think is best here.

By the way, Adam did add a flag to the config API endpoint that lets you surface API-set config as well.

Reply:
I meant as in, the configuration values of the webserver won't necessarily match the configuration values of a CLI instance. Therefore as a CLI user `fides view config` is useful for seeing what I have configured locally for my own instance. I wouldn't expect people to be running this on the webserver container, but that's an issue of documenting that expectation.

If I'm running locally and I run `fides view config`, I wouldn't expect to see values that are only configured on the webserver (especially since we don't required a webserver)

It sounds like we should document that for checking webserver configuration values specifically, one needs to use the API endpoint.